tests/e2e/helpers.go

```diff
@@ -77,8 +77,12 @@ func E2ELogf(t *testing.T, format string, args ...interface{}) {
 	t.Logf(fmt.Sprintf("%s: %s", time.Now().Format(time.RFC3339), format), args...)
 }
 
-func E2ELog(t *testing.T, format string) {
-	t.Log(fmt.Sprintf("%s: %s", time.Now().Format(time.RFC3339), format))
+func E2ELog(t *testing.T, args ...interface{}) {
+	t.Log(fmt.Sprintf("%s: %s", time.Now().Format(time.RFC3339), fmt.Sprint(args)))
+}
+
+func E2EErrorf(t *testing.T, format string, args ...interface{}) {
+	t.Errorf(fmt.Sprintf("E2E ERROR: %s: %s", time.Now().Format(time.RFC3339), format), args...)
```

Comment:
Mind switching this to `E2E-FAILURE` ?

That's what I used here https://github.com/openshift/compliance-operator/blob/e2caf33c12ae5ac06a4a5521b766841b01bdc055/tests/e2e/helpers.go#L611

:smile:

Reply:
Done